### PR TITLE
TC02 - User can't login with blank "Username" textbox

### PR DESCRIPTION
**Pull request by sattdn21-01-02**

### CODE REVIEW

File: src/test/java/com/logigear/LoginTest.java [hunk 1]
```diff
@@ -14,6 +14,21 @@
     private final HomePage homePage = new HomePage();
     private final LoginPage loginPage = new LoginPage();
 
+    @Description("TC02 - User can't login with blank Username textbox")
+    @Test
```
Comment (LoliSlayerMC) [inline src/test/java/com/logigear/LoginTest.java:18]: Use the wrong annotation. You should use this:
@test(description="...")

File: src/test/java/com/logigear/LoginTest.java [hunk 2]
```diff
@@ -14,6 +14,21 @@
...
+
+        Account account = new Account(Constant.BLANK_EMAIL, Constant.PASSWORD);
+        loginPage.login(account);
+
+        String actualMsg = loginPage.getGeneralErrorMessage();
+        String expectedMsg = Constant.FAIL_MSG_LOGIN_BLANK_INFORMATION;
+        Assert.assertEquals(actualMsg, expectedMsg);
+  }
+
     @Description("TC01 - User can log into Railway with valid username and password")
```
Comment (LoliSlayerMC) [inline src/test/java/com/logigear/LoginTest.java:32]: same here

**Review by LoliSlayerMC — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/LoginTest.java (hunk 1 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/LoginTest.java]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

**Review by LoliSlayerMC — APPROVED** [review]
LGTM

**Comment by LoliSlayerMC** [comment]
Resolve conflict please

File: src/test/java/com/logigear/LoginTest.java [hunk 3]
```diff
@@ -27,4 +27,18 @@ public void TC01() {
...
         Assert.assertEquals(actualMsg, expectedMsg);
     }
+
+    @Test(description = "TC02 - User can't login with blank Username textbox")
+    public void TC02() {
+        LoggerHelper.startTestCase("TC02 - User can't login with blank Username textbox");
+
+        homePage.goToLoginPage();
+
+        Account account = new Account(Constant.BLANK_EMAIL, Constant.PASSWORD);
```
Comment (tanmle) [inline src/test/java/com/logigear/LoginTest.java:37]: we don't need a contact for blank email

**Review by tanmle — CHANGES_REQUESTED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**